### PR TITLE
Create docs directory and QUICKSTART.md instructions; editorial pass …

### PR DESCRIPTION
**Pull request by richb-hanover**
…on README and BUILD-INSTALL

I like KeePassXC, but it was too hard to get started. I created a Quick Start document that tells how to install and get going with the pre-built binaries.

## Description

* Added docs folder
* Created docs/QUICKSTART.md
* Added some screen shots to illustrate Quick Start
* Renamed INSTALL.md to BUILD-INSTALL.md
* Other editorial changes to point newcomers to the Quick Start

## Types of changes

- ✅ Bug fix (non-breaking change which fixes an issue)
- ✅ New feature (non-breaking change which adds functionality)
- ✅ Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- ✅ I have read the **CONTRIBUTING** document. **[REQUIRED]**
- ✅ My code follows the code style of this project. **[REQUIRED]**
- ✅ All new and existing tests passed. **[REQUIRED]**
- ✅ I have compiled and verified my code with `-DWITH_ASAN=ON`. **[REQUIRED]**
- ✅ My change requires a change to the documentation and I have updated it accordingly.
- ✅ I have added tests to cover my changes.

### CODE REVIEW

File: BUILD-INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
+Build and Install KeePassXC
```
Comment (phoerious) [inline BUILD-INSTALL.md]: I think this file should still be called INSTALL.md. It's quite a standard name for many software packages.

File: docs/QUICKSTART.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Quick Start for KeePassXC
```
Comment (phoerious) [inline docs/QUICKSTART.md]: Maybe submit this document also to the website repository, so people can view it online? This file won't be packaged with binary releases.

File: BUILD-INSTALL.md (hunk 1 above)
Comment (richb-hanover) [reply BUILD-INSTALL.md]: I see your point. I've updated my PR to change the name back. Thanks.

File: docs/QUICKSTART.md (hunk 2 above)
Comment (richb-hanover) [reply docs/QUICKSTART.md]: That is a great idea. 

I wonder if any of the current team would like to update the KeePassXC.org site: I haven't worked on a Jekyll site in a while, and also don't fully understand the expectations for its organization. Thanks.

File: docs/QUICKSTART.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+This procedure gets KeePassXC running on your computer with browser integration, 
+using the pre-built binaries available for [download](https://keepassxc.org/download) 
+on the [KeePassXC site](https://keepassxc.org).
```
Comment (phoerious) [inline docs/QUICKSTART.md]: "on the" -> "from"

File: docs/QUICKSTART.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+**TL;DR** KeePassXC saves your passwords securely.
+When you double-click a URL in KeePassXC, it launches your default browser to that URL.
+With Browser Integration configured, KeePassXC automatically enters
```
Comment (phoerious) [inline docs/QUICKSTART.md]: "browser integration"

File: docs/QUICKSTART.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+When you double-click a URL in KeePassXC, it launches your default browser to that URL.
+With Browser Integration configured, KeePassXC automatically enters 
+username/password credentials into web page fields.
+
+## Installing and Starting KeePassXC
+
+* [Download the native installer](https://keepassxc.org/download) and install 
+KeePassXC for your  Windows, macOS, or Linux computer in the usual way for your platform.
+* Open the KeePassXC application.
+* Create a new database and give it a Master Key that's used to unlock the database file.
```
Comment (phoerious) [inline docs/QUICKSTART.md]: "master key"

File: docs/QUICKSTART.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+KeePassXC for your  Windows, macOS, or Linux computer in the usual way for your platform.
+* Open the KeePassXC application.
+* Create a new database and give it a Master Key that's used to unlock the database file. 
+This database holds entries (usernames, passwords, account numbers, notes) 
+for all your websites, programs, etc.
+* Create a few entries - enter the username, password, URL, and optionally notes about the entry.
+* KeePassXC securely stores those entries in the database.
+
+
+## To set up Browser Integration with KeePassXC:
```
Comment (phoerious) [inline docs/QUICKSTART.md]: "Setting up browser integration"

File: docs/QUICKSTART.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+* *Within KeePassXC*, go to **Tools->Settings** (on macOS, go to **KeePassXC->Preferences**.) 
+* In **Browser Integration**, check `Enable KeePassHTTP server`
```
Comment (phoerious) [inline docs/QUICKSTART.md]: Enable KeePassHTTP server also bold, not monospace.

File: docs/QUICKSTART.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+Leave the other options at their defaults.
+* *In your default web browser,* install the KeePassHTTP-Connector extension/add-on. Instructions for [[Firefox]](https://addons.mozilla.org/en-US/firefox/addon/keepasshttp-connector/?src=api) or [[Chrome]](https://chrome.google.com/webstore/detail/keepasshttp-connector/dafgdjggglmmknipkhngniifhplpcldb?utm_source=chrome-app-launcher-info-dialog)
```
Comment (phoerious) [inline docs/QUICKSTART.md]: No square brackets around browser links.

File: docs/QUICKSTART.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+
+* *Within KeePassXC*, go to **Tools->Settings** (on macOS, go to **KeePassXC->Preferences**.) 
+* In **Browser Integration**, check `Enable KeePassHTTP server`
+Leave the other options at their defaults.
+* *In your default web browser,* install the KeePassHTTP-Connector extension/add-on. Instructions for [[Firefox]](https://addons.mozilla.org/en-US/firefox/addon/keepasshttp-connector/?src=api) or [[Chrome]](https://chrome.google.com/webstore/detail/keepasshttp-connector/dafgdjggglmmknipkhngniifhplpcldb?utm_source=chrome-app-launcher-info-dialog) 
+* Click the KeePassXC icon in the upper-right corner. You'll see the dialog below. 
+* Click the blue Connect button to make the browser extension connect to the KeePassXC application. 
+<img src="./KeePassXC-Connect.png" height="200" alt="KeePassXC Connect dialog">
+* *Switch back to KeePassXC.* You'll see a dialog (below) indicating that a request to connect has arrived. 
+* Give the connection a name (perhaps `Keepass-Browsername`, any unique name will suffice) and click OK to accept it.
```
Comment (phoerious) [inline docs/QUICKSTART.md]: Italics, no monospace.

File: docs/QUICKSTART.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+* This one-time operation connects KeePassXC and your browser.
+<img src="./KeePassXC-Accept-Button.png" height="200" alt="KeePassXC accept connection dialog">
+
+## To use Browser Integration
```
Comment (phoerious) [inline docs/QUICKSTART.md]: "Using browser integration"

File: docs/QUICKSTART.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+
+* *Within the KeePassXC application,* double-click the URL of an entry,
```
Comment (phoerious) [inline docs/QUICKSTART.md]: "Within KeePassXC"

File: docs/QUICKSTART.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+or select it and type Ctl-U (Cmd-U on macOS).
```
Comment (phoerious) [inline docs/QUICKSTART.md]: Ctrl+U, Cmd+U

File: docs/QUICKSTART.md [hunk 13]
```diff
@@ -0,0 +1,46 @@
...
+* Your browser opens to that URL.
+* If there are Username/Password fields on that page, you will see the dialog below.
```
Comment (phoerious) [inline docs/QUICKSTART.md]: username/password

File: docs/QUICKSTART.md [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+Click "Allow" to confirm the KeePassXC access the credentials and to auto-fill the fields. 
+* Check the "Remember this decision" to allow this each time you visit the page.
```
Comment (phoerious) [inline docs/QUICKSTART.md]: No the after Check

File: docs/QUICKSTART.md [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+* This one-time operation connects KeePassXC and your browser.
+<img src="./KeePassXC-Accept-Button.png" height="200" alt="KeePassXC accept connection dialog">
+
+## To use Browser Integration
+
+* *Within the KeePassXC application,* double-click the URL of an entry,
+or select it and type Ctl-U (Cmd-U on macOS).
+* Your browser opens to that URL.
+* If there are Username/Password fields on that page, you will see the dialog below.
+Click "Allow" to confirm the KeePassXC access the credentials and to auto-fill the fields.
```
Comment (phoerious) [inline docs/QUICKSTART.md]: "Click "Allow" to confirm that KeePassXC may access the credentials to auto-fill the fields"

**Comment by richb-hanover** [comment]
Woo woo! Thanks!

File: docs/QUICKSTART.md [hunk 16]
```diff
@@ -2,45 +2,46 @@
...
 * This one-time operation connects KeePassXC and your browser.
 <img src="./KeePassXC-Accept-Button.png" height="200" alt="KeePassXC accept connection dialog">
 
-## To use Browser Integration
+## Using Browser Integration
 
-* *Within the KeePassXC application,* double-click the URL of an entry,
-or select it and type Ctl-U (Cmd-U on macOS).
+* *Within KeePassXC,* double-click the URL of an entry,
+or select it and type Ctl+U (Cmd+U on macOS).
```
Comment (phoerious) [inline docs/QUICKSTART.md]: You missed the missing r here. ;-)

**Comment by richb-hanover** [comment]
And the PR has been updated, too. Thanks again.

File: docs/QUICKSTART.md (hunk 16 above)
Comment (richb-hanover) [reply docs/QUICKSTART.md]: Err... I'm blanking on this - "missing r"?

Comment (phoerious) [reply docs/QUICKSTART.md]: Ctrl, not Ctl.

Comment (richb-hanover) [reply docs/QUICKSTART.md]: D'oh. That's a Mac-ism, I think. (Or I blame my Mac-centric view.) Good catch.

**Comment by phoerious** [comment]
Thanks!

**Pull request merged**